jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/NoOpSessionManager.java

```diff
@@ -0,0 +1,237 @@
+  public HttpSession newHttpSession(HttpServletRequest request) {
+
+    String id = _idManager.newSessionId(request, System.currentTimeMillis());
+    String nodeId = _idManager.getNodeId(id, request);
+    return new NoOpSession(id, nodeId, _context);
+  }
+
+  @Override
+  public boolean getHttpOnly() {
+    return false;
+  }
+
+  @Override
+  public int getMaxInactiveInterval() {
+    return -1;
+  }
+
+  @Override
+  public void setMaxInactiveInterval(int seconds) {
+  }
+
+  @Override
+  public void setSessionHandler(SessionHandler handler) {
+    _handler = handler;
+  }
+
+  @Override
+  public void addEventListener(EventListener listener) {
+  }
+
+  @Override
+  public void removeEventListener(EventListener listener) {
+
+  }
+
+  @Override
+  public void clearEventListeners() {
+  }
+
+  @Override
+  public HttpCookie getSessionCookie(HttpSession session, String contextPath,
+                                     boolean requestIsSecure) {
+    return null;
+  }
+
+  @Override
+  public SessionIdManager getSessionIdManager() {
+    return _idManager;
+  }
+
+  @Override
+  public SessionIdManager getMetaManager() {
+    return getSessionIdManager();
+  }
+
+  @Override
+  public void setSessionIdManager(SessionIdManager idManager) {
+    _idManager = idManager;
+  }
+
+  @Override
+  public boolean isValid(HttpSession session) {
+    return ((NoOpSession) session).isValid();
+  }
+
+  @Override
+  public String getNodeId(HttpSession session) {
+    return ((NoOpSession) session).getNodeId();
+  }
+
+  @Override
+  public String getClusterId(HttpSession session) {
+    return ((NoOpSession) session).getId();
+  }
+
+  @Override
+  public HttpCookie access(HttpSession session, boolean secure) {
+    return null;
+  }
+
+  @Override
+  public void complete(HttpSession session) {
+    ((NoOpSession) session).complete();
+  }
+
+  @Override
+  public void setSessionIdPathParameterName(String parameterName) {
+  }
+
+  @Override
+  public String getSessionIdPathParameterName() {
+    return null;
+  }
+
+  @Override
+  public String getSessionIdPathParameterNamePrefix() {
+    return null;
+  }
+
+  @Override
+  public boolean isUsingCookies() {
+    return false;
+  }
+
+  @Override
+  public boolean isUsingURLs() {
+    return false;
+  }
+
+  @Override
+  public Set<SessionTrackingMode> getDefaultSessionTrackingModes() {
+    return null;
+  }
+
+  @Override
+  public Set<SessionTrackingMode> getEffectiveSessionTrackingModes() {
+    return null;
+  }
+
+  @Override
+  public void setSessionTrackingModes(Set<SessionTrackingMode> sessionTrackingModes) {
+  }
+
+  @Override
+  public SessionCookieConfig getSessionCookieConfig() {
+    return null;
+  }
+
+  @Override
+  public boolean isCheckingRemoteSessionIdEncoding() {
+    return true;
+  }
+
+  @Override
+  public void setCheckingRemoteSessionIdEncoding(boolean remote) {
+  }
+
+  @Override
+  public void renewSessionId(String oldClusterId, String oldNodeId, String newClusterId,
```

Comment:
all params should be wrapped in one line. If that lines does not fit wrapping should be each param in its own line +4 spaces. the google-java-formatter should do that.